_validate/validate.py

```diff
@@ -23,9 +23,9 @@
 
 
 JSON_SCHEMA = os.path.join(os.path.dirname(__file__), "addonVersion_schema.json")
-JsonObjT = typing.Dict[str, typing.Any]
-
+VER_URL = "https://raw.githubusercontent.com/nvaccess/addon-datastore-transform/main/nvdaAPIVersions.json"
```

Comment:
i don't think we should download a copy like this.
Instead, the dependency should be passed in via command line e.g. `runvalidate <pathToAddonMetadataFile.json> <pathTonvdaAPIVersions.json>`